simplq/src/components/pages/Join/Form.jsx

```diff
@@ -23,9 +24,25 @@ export function JoinQueueForm(props) {
     }
   }
 
-  function handleContactChange(e) {
-    setContact(e);
-    setInvalidContact(false);
+  function handleContactChange(value, country) {
+    setContact(value);
+    const phoneUtil =  PhoneNumberUtil.getInstance();
+
+    if (country != null) {
+      // to make sure that the number is parsed as an international number, prepend +. 
+      const phoneNr = '+' + value;
+
+      try {
+        const isValidNumber = phoneUtil.isValidNumberForRegion(phoneUtil.parse(phoneNr, country.countryCode), country.countryCode);
+        if (isValidNumber) {
+        }
+        setInvalidContact(!isValidNumber);
```

Comment:
This if is not required

Reply:
Yeah, you're right, I forgot to remove it. I updated the code.